Source/Core/Core/Boot/Boot.h

```diff
@@ -116,7 +116,7 @@ class CBoot
   static bool Load_BS2(const std::string& boot_rom_filename);
 
   static void SetupGCMemory();
-  static bool SetupWiiMemory();
+  static bool SetupWiiMemory(bool devkit);
```

Comment:
What about making this take console type instead?

If not and we stay on devkit vs. non devkit, please make it an enum for readability.